integration/raft/channel_participation_test.go

```diff
@@ -60,29 +63,34 @@ var _ = Describe("ChannelParticipation", func() {
 		os.RemoveAll(testDir)
 	})
 
-	Describe("basic etcdraft network without a system channel", func() {
+	Describe("three node etcdraft network without a system channel", func() {
+		startOrderer := func(o *nwo.Orderer) {
+			ordererRunner := network.OrdererRunner(o)
+			ordererProcess := ifrit.Invoke(ordererRunner)
+			Eventually(ordererProcess.Ready, network.EventuallyTimeout).Should(BeClosed())
+			Eventually(ordererRunner.Err(), network.EventuallyTimeout).Should(gbytes.Say("Registrar initializing without a system channel, number of application channels: 0"))
+			ordererProcesses = append(ordererProcesses, ordererProcess)
+			ordererRunners = append(ordererRunners, ordererRunner)
+		}
+
 		BeforeEach(func() {
-			raftConfig := nwo.BasicEtcdRaft()
-			network = nwo.New(raftConfig, testDir, client, StartPort(), components)
+			network = nwo.New(nwo.MultiNodeEtcdRaft(), testDir, client, StartPort(), components)
 			network.ChannelParticipationEnabled = true
 			network.BootstrapMethod = "none"
 			network.GenerateConfigTree()
 			network.Bootstrap()
+		})
 
-			orderer := network.Orderer("orderer")
-			ordererRunner = network.OrdererRunner(orderer)
-			ordererProcess = ifrit.Invoke(ordererRunner)
-			Eventually(ordererProcess.Ready, network.EventuallyTimeout).Should(BeClosed())
-			Eventually(ordererRunner.Err(), network.EventuallyTimeout).Should(gbytes.Say("Registrar initializing without a system channel, number of application channels: 0"))
+		It("starts an orderer but rejects channel creation requests via the legacy channel creation", func() {
+			orderer1 := network.Orderer("orderer1")
+			startOrderer(orderer1)
 
-			nwo.ChannelParticipationList(network, orderer, nil)
-		})
+			nwo.ChannelParticipationList(network, orderer1, nil)
```

Comment:
What is this? Why is it in nwo?

Looking at the implementation, it only makes assertions. It's not part of the network abstraction but tests. I do not believe it belongs in nwo.